Release v10.0.0 (#242)
paraswap/paraswap-sdk: README.md

Refer to the documentation of the Velora API: https://developers.velora.xyz/

## Features

> **Versatility**: works with [web3](https://www.npmjs.com/package/web3) or [ethers](https://www.npmjs.com/package/ethers) without direct dependency

Reviewer: Don't drop `viem`
... web3 or ethers without direct dependency, or with locally-sourced all-organic free-range viem
or something.
Important to mention it works, even with a dependency

> **Versatility**: works with [web3](https://www.npmjs.com/package/web3) or [ethers](https://www.npmjs.com/package/ethers) without a direct dependency — or with locally-sourced, all-organic, free-range [viem](https://viem.sh/) (which ships with the SDK)

Reviewer: Please don't, that was a joke
```suggestion
**Versatility**: works with [web3](https://www.npmjs.com/package/web3) or [ethers](https://www.npmjs.com/package/ethers) without a direct dependency — or with [viem](https://viem.sh/) (also used internally for its utils)
```

`which ships with SDK` can be confused for `you don't need to install/init it locally`

### Delta is now v2 — orders are built server-side

- **You no longer build the order locally.** Pass the quoted `route` + `side` (from `getDeltaPrice` / `quote.delta`) to `buildDeltaOrder` / `submitDeltaOrder`; the server returns the typed data to sign. You no longer pass local amounts.
- **One signer for every family.** `signDeltaOrder(builtOrder)` signs Standard, External, TWAP Sell and TWAP Buy orders. The old per-family signers **`signExternalDeltaOrder`** and **`signTWAPDeltaOrder`** are removed.
- **`getDeltaPrice` returns a route-based `DeltaPrice`** (`route` + `alternatives`). The old `BridgePrice` type is gone — cross-chain is handled in-route via `destChainId`.
- **`getBridgeInfo` → `getBridgeRoutes`** (now returns a flat `BridgeRoute[]`), plus a new `getBridgeProtocols`.
- **New `getAgentsList`** (`sdk.delta.getAgentsList`).
- **New read-only order types** surface through the read paths: `ProductiveOrder` (server-managed, no builder) and `FillableOrder` (a `partiallyFillable` Standard order — treat the same as `Order`).
- All Delta endpoints now use the `/v2/delta/...` prefix.

### `limitOrders` → `otcOrders`

> The limit-orders module was renamed to OTC orders. Update both the namespace and the constructors:

Reviewer: Not really. deprecated legacy (not Delta) limit order methods removed, otc methods remain under their own name
